### ISSUE TITLE
Could not find a declaration file for module 'mui-chips-input'

### ISSUE THREAD

**Issue by FairouzBF** [viclafouch/mui-chips-input#34]
**Describe the bug**
The package works just fine, i can use the chips but when importing  { MuiChipsInput as ChipInput } from 'mui-chips-input', the name of the package is underlined with red and i get the following message : 
```
Could not find a declaration file for module 'mui-chips-input'. '/Users/UBENFFA/workspace/tl2-schedule-management-subsystem/schedule-management-service/apps/web/schedule-management-ui/node_modules/mui-chips-input/dist/mui-chips-input.es.js' implicitly has an 'any' type.
  There are types at '/Users/UBENFFA/workspace/tl2-schedule-management-subsystem/schedule-management-service/apps/web/schedule-management-ui/node_modules/mui-chips-input/dist/index.d.ts', but this result could not be resolved when respecting package.json "exports". The 'mui-chips-input' library may need to update its package.json or typings.
```

**Expected behavior**

**Screenshots**
![Screenshot 2023-07-12 at 15 44 43](https://github.com/viclafouch/mui-chips-input/assets/60386124/43670dfd-6490-4798-8873-0d0e980e53fc)

**Desktop (please complete the following information):**
System:
    OS: macOS 12.6.7
  Binaries:
    Node: 14.21.1 - ~/.nvm/versions/node/v14.21.1/bin/node
    Yarn: Not Found
    npm: 6.14.17 - ~/.nvm/versions/node/v14.21.1/bin/npm
  Browsers:
    Chrome: 114.0.5735.198
    Edge: Not Found
    Safari: 16.5.1
  npmPackages:
    @emotion/react:  11.11.1 
    @emotion/styled:  11.11.0 
    @mui/base:  5.0.0-beta.7 
    @mui/core-downloads-tracker:  5.14.0 
    @mui/icons-material: ^5.14.0 => 5.14.0 
    @mui/lab: ^5.0.0-alpha.136 => 5.0.0-alpha.136 
    @mui/material: ^5.14.0 => 5.14.0 
    @mui/private-theming:  5.13.7 
    @mui/styled-engine:  5.13.2 
    @mui/system:  5.14.0 
    @mui/types:  7.2.4 
    @mui/utils:  5.13.7 
    @mui/x-date-pickers: ^6.9.2 => 6.9.2 
    @mui/x-date-pickers-pro: ^6.9.2 => 6.9.2 
    @mui/x-license-pro:  6.9.0 
    @types/react: ^18.0.25 => 18.0.25 
    react: ^18.2.0 => 18.2.0 
    react-dom: ^18.2.0 => 18.2.0 
    typescript: ^5.1.6 => 5.1.6

**Comment by airalpha** [comment]
I have a same error.
How to fix this please ?

**Comment by viclafouch** [comment]
Hello ! Do you try with the version `2.1.2` plz :)

**Comment by airalpha** [comment]
> Hello ! Do you try with the version `2.1.2` plz :)

The issue is resolved in version `2.1.2`. Thanks!
